### PR TITLE
Ramy Bachayev

### PR DESCRIPTION
**Pull request by ramyb3**
---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1202137838393549

### CODE REVIEW

File: components/header.tsx [hunk 1]
```diff
@@ -3,41 +3,56 @@ import { CommonType } from '@/lib/interface'
...
-          </ul>
+          <Image
+            src="/icons/veahavta-icon.svg"
+            height={'62px'}
+            width={'142px'}
+          />
+
+          <div className="flex flex-cols-2 justify-between place-items-center">
+            <ul className="hidden md:flex flex-cols-4 gap-8 text-2xl">
+              {data.appLinks.map((x, index) => {
```
Comment (omrirz) [inline components/header.tsx]: why "x" and not "appLink"?

File: components/home-components/contact-us.tsx [hunk 2]
```diff
@@ -1,7 +1,74 @@
...
+            text={data.homepage.contactUsHeading}
+          />
+
+          <h2 className="text-4xl font-bold">{data.homepage.contactUsTitle}</h2>
+          <p className="text-xl max-w-[400px] pt-3">
+            {data.homepage.contactUsText}
+          </p>
+
+          <ul className="grid gap-2.5 lg:grid-cols-auto-1fr grid-rows-auto-1fr">
+            {data.common.contactUsLinks.map((x, index) => {
```
Comment (omrirz) [inline components/home-components/contact-us.tsx]: why "x" and not "contactUsLink"?

File: components/home-components/what-we-do.tsx [hunk 3]
```diff
@@ -1,7 +1,34 @@
...
+  return (
+    <WrapperLarge className="bg-brown-bg place-items-center gap-1.5 text-center">
+      <div className="text-2xl text-red py-4">
+        {data.homepage.whatWeDoHeading}
+      </div>
+
+      <h2 className="text-7xl font-bold">{data.homepage.whatWeDoTitle}</h2>
+
+      <div className="grid mt-10 xl:grid-cols-4 gap-5 md:grid-cols-2 gap-5">
+        {data.homepage.whatWeDoCards.map((x, index) => {
```
Comment (omrirz) [inline components/home-components/what-we-do.tsx]: why "x" and not "whatWeDoCard" or "card"?

File: components/data-components/contact-form.tsx [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+          error={formik.errors.lastName}
+        />
+        <Input
+          name="email"
+          text={data.common.contactUsFormEmail}
+          formik={formik}
+          touch={formik.touched.email}
+          error={formik.errors.email}
+        />
+        <Input
```
Comment (omrirz) [inline components/data-components/contact-form.tsx:72]: This should be of type tel so on mobile you will get the numpad keyboard

File: components/data-components/contact-form.tsx [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+          error={formik.errors.firstName}
+        />
+        <Input
+          name="lastName"
+          text={data.common.contactUsFormLastName}
+          formik={formik}
+          touch={formik.touched.lastName}
+          error={formik.errors.lastName}
+        />
+        <Input
```
Comment (omrirz) [inline components/data-components/contact-form.tsx:64]: Should be of type email so you'll get the email keyboard on mobile

**Review by omrirz — CHANGES_REQUESTED** [review]
Overall it looks good. There are a few changes needed.

1. The footer email link is broken.
Screen Shot 2022-04-22 at 22 42 07

2. Sections width is not the same:
Screen Shot 2022-04-22 at 22 42 47

3. on mobile (320px) there are overflow x issues.
Screen Shot 2022-04-23 at 0 40 34

4. Why delete .env.example? other developers working with you on the project need it as well.

File: components/data-components/contact-form.tsx (hunk 5 above)
Comment (ramyb3) [reply components/data-components/contact-form.tsx:64]: I fixed that.

File: components/data-components/contact-form.tsx (hunk 4 above)
Comment (ramyb3) [reply components/data-components/contact-form.tsx:72]: I fixed that.

File: components/home-components/what-we-do.tsx (hunk 3 above)
Comment (ramyb3) [reply components/home-components/what-we-do.tsx]: I fixed that.

File: components/home-components/contact-us.tsx (hunk 2 above)
Comment (ramyb3) [reply components/home-components/contact-us.tsx]: I fixed that.

File: components/header.tsx (hunk 1 above)
Comment (ramyb3) [reply components/header.tsx]: I fixed that.

**Comment by ramyb3** [comment]
1.  I fixed that.
2.  I fixed that.
3.  I fixed that.
4.  This is something I didn't think of, I've added the file.

**Pull request closed without merging**